scripts/ci/crates.py

```diff
@@ -280,9 +320,9 @@ def publish(dry_run: bool, token: str) -> None:
     ctx.finish(dry_run)
 
     if not dry_run:
+        env = {**os.environ.copy(), "RERUN_IS_PUBLISHING": "yes", "IS_IN_RERUN_WORKSPACE": "yes"}
```

Comment:
I don't think we should set `IS_IN_RERUN_WORKSPACE` here. That is meant for detecting a developer (us) working locally.